aircraft_heirs.py

```diff
@@ -0,0 +1,171 @@
+        super().__init__(aircraft_id, 'narrow_body_plane', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+
+    def apply_aircraft_feature(self):
+        print('Applying Narrow-body plane Aircraft feature')
+
+
+class Cargo(CargoAircraft):
+    def __init__(self, aircraft_id, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, 'cargo', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+
+    def apply_aircraft_feature(self):
+        print('Applying Cargo Aircraft feature')
+
+
+class MaritimePatrol(Aircraft, WaterLandingMixin):
+    def __init__(self, aircraft_id, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, 'maritime_patrol', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+        self.is_marine = True
+
+    def apply_aircraft_feature(self):
+        print('Applying Maritime Patrol Aircraft feature')
+
+
+class WaterPlane(Aircraft, WaterLandingMixin):
+    def __init__(self, aircraft_id, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, 'water_plane', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+        self.is_marine = True
+
+    def apply_aircraft_feature(self):
+        print('Applying Water-plane Aircraft feature')
+
+
+class Helicopter(Aircraft):
+    def __init__(self, aircraft_id, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, 'helicopter', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+
+    def apply_aircraft_feature(self):
+        print('Applying Helicopter Aircraft feature')
+
+
+class Fighter(Aircraft, BombThrowingMixin):
+    def __init__(self, aircraft_id, width, landing_length, num_of_seats,
+                 destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates):
+        super().__init__(aircraft_id, 'fighter', width, landing_length, num_of_seats,
+                         destination_coordinates, departure_coordinates, departure_time, speed, actual_coordinates)
+        self.is_combat = True
```

Comment:
you can define if the aircraft is combat by checking if the class is subclass of mixin